cli/cmds/cluster_create.go

```diff
@@ -196,6 +196,17 @@ func newCluster(name, namespace string, config *CreateConfig) *v1alpha1.Cluster
 				StorageClassName:   ptr.To(config.storageClassName),
 				StorageRequestSize: config.storageRequestSize,
 			},
+			Sync: v1alpha1.SyncConfig{
+				ConfigMaps: v1alpha1.SyncConfigOptions{
+					Enabled: true,
+				},
+				Secrets: v1alpha1.SyncConfigOptions{
+					Enabled: true,
+				},
+				Services: v1alpha1.SyncConfigOptions{
+					Enabled: true,
+				},
+			},
```

Comment:
I would drop this, and leverage the default values.

Reply:
yeah for some reason, the cli wont respect the default values

Reply:
Hm, that's strange. Maybe it's because of some `omitempty` or default kubebuilder tags? Does it work sending the yaml with an empty sync?